exercism/annalyns-infiltration/LincolnInitialSolution.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+
+static class QuestLogic
+{
+    public static bool CanFastAttack(bool knightIsAwake)
+    {
+        if(knightIsAwake){
+            return false;
+        }
+        else{
+            return true;
+        }
+    }
```

Comment:
When you see 
```cs
if (someCondition) {
  return true;
} else {
  return false;
}
```
or vice versa, you can almost always skip the conditionals. Remember, booleans are already booleans. So this function can be simplified to

```suggestion
    public static bool CanFastAttack(bool knightIsAwake)
    {
        return !knightIsAwake;
    }
```